db/migrate/20150817204307_add_nullify_to_form_original_id_fk.rb

```diff
@@ -0,0 +1,6 @@
+class AddNullifyToFormOriginalIdFk < ActiveRecord::Migration
+  def change
+    remove_foreign_key "forms", column: "original_id", name: "forms_standard_id_fk"
+    add_foreign_key "forms", "forms", column: "original_id", name: "forms_standard_id_fk", on_delete: :nullify
```

Comment:
Please add the nullify for all original_id FKs. I believe there are three in all.